.eslintrc.json

```
@@ -9,6 +9,8 @@
     "airbnb-typescript/base"
   ],
   "rules": {
+    "import/prefer-default-export": 0,
+    "import/no-default-export":  2,
```

TIL you can [use numbers to configure rules](https://eslint.org/docs/user-guide/configuring#configuring-rules)! However, I find the numbers a bit confusing; can we use the names, instead?

```suggestion
    "import/prefer-default-export": "off",
    "import/no-default-export":  "error",
```

---

Oh that's way better (and TIL you can use names lol)